### DISCUSSION TITLE
IntelliSense support for assembly files

### DISCUSSION THREAD

**Discussion by jasr93** [microsoft/vscode-cpptools#11141] [Q&A] [answered]
Hello,
I'm working on a project related to embedded software using cross compilers and mixing C/CPP and ASM (assembly) files.
My build system is based on CMake and IntelliSense can parse compile_commands.json to index C and CPP files.
I can see that the ASM files (*.s or *.S) are listed correctly in compile_commands.json, however they are not parsed by IntelliSense and they remain inactive.
Any ideas on how I can include these ASM files for indexing?

**Comment by sean-mcmanus** [comment, marked as answer]
Our extension only parses symbols and handles IntelliSense for C/C++ and not assembly code files.

**Reply by jasr93** [reply]
Any plan to support assembly files?

**Reply by sean-mcmanus** [reply]
Not currently.
